system-addon/data/content/tippytop/top_sites.json

```diff
@@ -46,28 +26,23 @@
   },
   {
     "title": "ebay",
-    "urls": ["https://www.ebay.com", "https://www.ebay.co.uk/"],
+    "urls": ["https://www.ebay.com", "https://www.ebay.co.uk/", "https://ebay.de"],
```

Comment:
Same here. Should we include "https://ebay.ca/"?

https://github.com/mozilla/tippy-top-sites/blob/df4de4b322d3cf0742b6b1527a15bf9f291411d5/top_sites.json#L389

Reply:
ebay isn't a default site for Canada, so that's why it wasn't included.